jsFiles/utility.js

```diff
@@ -0,0 +1,73 @@
+// displayWeatherUserLocation() is used to get the location of the user's computer: latitude and longitude
```

Comment:
If you're writing comment for a function you can use block comments, and you can specify what is the type of arguments in that function and add description to it as well:
https://jsdoc.app/about-getting-started.html

```js
/**
 * displayWeatherUserLocation() is used to get the location of the user's computer: latitude and longitude
 * (geographical coordinates). These data is used to build the url for Open Weather API,
 * where the data of the weather data is fetch from.
 *
 * @param {Number} pos - user position coordinates
 */
function displayWeatherUserLocation(pos) {
  const latitude = pos.coords.latitude.toFixed(2);
  const longitude = pos.coords.longitude.toFixed(2);
  const apiUrl = `${apiRootUrl}weather?lat=${latitude}&lon=${longitude}&appid=${KEY}&units=metric`;
  fetchData(apiUrl);
}
```